package.json

```diff
@@ -21,10 +21,14 @@
         "node": ">=10"
     },
     "dependencies": {
+        "@types/node": "^18.15.11",
+        "ipinfo-express": "^1.1.0",
         "node-ipinfo": "^3.0.2"
     },
     "devDependencies": {
+        "@types/express": "^4.17.17",
         "dotenv": "^8.2.0",
-        "prettier": "^2.2.1"
+        "prettier": "^2.2.1",
```

Comment:
Oh we already have prettier installed... then we need to run it. See `scripts/fmt.sh` in this repo.